assignments/arrays.js

```diff
@@ -63,34 +63,66 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
-
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === 34 - 1)  {
+      // console.log(inventory[i]); 
+      var make = inventory[i].car_make;
+      var model = inventory[i].car_model;
+      var year = inventory[i].car_year;
+    }
+  }
+  
+  console.log(`Car 33 is a ${year} ${make} ${model}` );
+  
+  
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.slice(inventory.length -1);
+console.log(lastCar[0].car_make, lastCar[0].car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i = 0; i < inventory.length; i++ ) {
+  let car = inventory[i].car_model.charAt(0).toUpperCase() + inventory[i].car_model.slice(1);
+  carModels.push(car);
+}
+console.log(carModels.sort());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (let i = 0; i < inventory.length; i++ ) {
+  carYears.push(inventory[i].car_year);
+}
+console.log(carYears.sort());
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars =[];
-console.log(); 
+for(i = 0; i < carYears.length; i++) {
+  if (carYears[i] < 2000) {
+    continue
+  } else {
+    oldCars.push(carYears[i])
+  }
+}
+console.log(oldCars.length); 
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi =[];
-console.log();
+for(let i = 0; i < inventory.length; i++) {
+  if(inventory[i].car_make === 'Audi' ) {
+    BMWAndAudi.push(inventory[i]);
+  }else if (inventory[i].car_make === 'BMW') {
 
+   BMWAndAudi.push(inventory[i]);
+  } else {
+    continue
```

Comment:
Tip: You don't necessarily need an `else` and `continue` here. If the item doesn't meet the 2 conditions you set above then it will automatically continue.